lib/style_parser/filterer/index.js

```diff
@@ -20,16 +20,30 @@ var css = require('css');
 function filter(cssInput, cb) {
   // Remove HTML comment start/end
   cssInput = removeHtmlComments(cssInput);
+  cssInput = handleParsingErrors(cssInput);
```

Comment:
handleParsingErrors parses the input css which might have errors.
we will have to parse it second time on css which has errors removed, so that the parsed sheet has no parsing errors.